jta-and-hibernate-standalone/README.md

```diff
@@ -0,0 +1,28 @@
+Standalone JTA with Hibernate example.
+==================================================================================================
+Author: Gytis Trikleris;
+Level: Advanced;
+Technologies: JTA, CDI, JPA
+
+What is it?
+-----------
+
+This example demonstrates how to use JTA with Hibernate in standalone application.
+
+
+System requirements
+-------------------
+
+All you need to build this project is Java 8 (Java SDK 1.8) or better and Maven 3.3.3 or better.
+
+
+Build and Deploy the Quickstart
+-------------------------------
+
+1. Open a command line and navigate to the root directory of this quickstart.
+2. Execute quickstart:
+
+        mvn clean test
+        
+
+NOTE: there is a known issue with hibernate failing to close the connection during the afterCompletion step. (https://issues.jboss.org/browse/JBTM-2676)
```

Comment:
All quickstarts need to follow the standard format. I believe the following was the recommended layout:

## OVERVIEW
## USAGE
## EXPECTED OUTPUT
## WHAT JUST HAPPENED?